src/sentry/eventstore/snuba/backend.py

```diff
@@ -426,7 +426,9 @@ def get_adjacent_event_ids(self, event, filter):
         prev_filter = deepcopy(filter)
         prev_filter.conditions = prev_filter.conditions or []
         prev_filter.conditions.extend(get_before_event_condition(event))
-        prev_filter.start = datetime.fromtimestamp(0)
+
+        # We only store 90 days of data, add a few extra days just in case
+        prev_filter.start = event.datetime - timedelta(days=100)
```

Comment:
this is quite a few days, I don't think I quite get why we need to go back so far

Reply:
Neither do I, but I didn't want to break any existing assumptions. The goal was to have this feature continue to work the way it was before.